core/services/job/orm.go

```diff
@@ -275,6 +275,19 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 				return errors.Wrap(err, "failed to create BlockhashStore spec")
 			}
 			jb.BlockhashStoreSpecID = &specID
+		case Bootstrap:
```

Comment:
I think we're missing a corresponding change to the DeleteJob orm. It'd be great to add a test in job_orm_test.go which for all job types does a add/delete/add and asserts the db looks good after each